api/docs/release.dox

```diff
@@ -182,6 +182,9 @@ Further non-compatibility-affecting changes include:
    reg_is_strictly_zmm().
  - Added the function reg_is_avx512_extended().
  - Added the function instr_is_opmask().
+ - Added the functions instr_set_enc_hint(), instr_is_enc_hint().
```

Comment:
Not sure about using "enc": spelling it out is clearer and avoids any confusion.  I would do the same for everything: instr_set_encoding_hint, dr_encoding_hint_t, etc.

Reply:
s/enc_hint/encoding_hint/